src/webgpu/api/validation/buffer/mapping.spec.ts

```diff
@@ -943,7 +943,7 @@ g.test('getMappedRange,disjointRanges')
     t.testGetMappedRangeCall(success, buffer, offset2, size2);
   });
 
-g.test('getMappedRange,disjoinRanges_many')
+g.test('getMappedRange,disjointRanges_many')
```

Comment:
Generally I'd say this should be `disjoint_ranges_many` or `disjoint_ranges,many` since "disjointRanges" doesn't refer to a camelCase thing from the API spec...
However that is not a very important rule and we'd have to update  _two_ tests to fix that so LGTM